packages/runtime-core/src/components/BaseTransition.ts

```diff
@@ -43,14 +43,14 @@ export interface BaseTransitionProps<HostElement = RendererElement> {
   onLeaveCancelled?: (el: HostElement) => void // only fired in persisted mode
 }
 
-export interface TransitionHooks {
+export interface TransitionHooks<HostElement = RendererElement> {
```

Comment:
```suggestion
export interface TransitionHooks<HostElement extends RendererElement = RendererElement> {
```
To keep the same typing as before

Reply:
Sorry, just checked other places and there's a few `HostElement = RendererElement`, probably is the way it was is better, I'm sorry about that.